src/plugins/workspace/public/utils.ts

```diff
@@ -108,14 +109,18 @@ export function isAppAccessibleInWorkspace(app: App, workspace: WorkspaceObject)
   return false;
 }
 
+// Get all apps that should be displayed in workspace when create/update a workspace.
 export const filterWorkspaceConfigurableApps = (applications: PublicAppInfo[]) => {
   const visibleApplications = applications.filter(({ navLinkStatus, chromeless, category, id }) => {
-    return (
+    const filterCondition =
       navLinkStatus !== AppNavLinkStatus.hidden &&
       !chromeless &&
-      !DEFAULT_SELECTED_FEATURES_IDS.includes(id) &&
-      category?.id !== DEFAULT_APP_CATEGORIES.management.id
-    );
+      !DEFAULT_SELECTED_FEATURES_IDS.includes(id);
+    // If the category is management, only retain dashboards management.
+    if (category?.id === DEFAULT_APP_CATEGORIES.management.id) {
+      return filterCondition && id === MANAGEMENT_APP_ID;
+    }
+    return filterCondition;
```

Comment:
can we add more comments of why we would like add an exception case for management

Reply:
Yes, the comments has been added.